### ISSUE TITLE
How to avoid section's value can't be zero?

### ISSUE THREAD

**Issue by mrdc** [imaNNeoFighT/fl_chart#640]
Hi, 

After upgrade to version 0.35.0 I receive `section's value can't be zero` (which is expected as it's breaking change from 0.30.0 to 0.35.0), because in some cases `section.value` is `0`. 
What should I do if I want to show 0 value on chart? 
`FlChart 0.30.0`:
![Screenshot 2021-04-14 at 18 39 22](https://user-images.githubusercontent.com/16876990/114738689-dfa4dc00-9d50-11eb-95bc-2b67b93c4896.png)
For example, I use a chart (https://github.com/imaNNeoFighT/fl_chart/blob/master/example/lib/pie_chart/samples/pie_chart_sample2.dart) with 2 predefined sections and one section can be 0.
If I use ternary operation `value: myValue != 0 ? myValue : null`, the error goes away, but chart's look is incorrect in this case (code below):
![Screenshot 2021-04-14 at 18 14 42](https://user-images.githubusercontent.com/16876990/114736939-37424800-9d4f-11eb-92dc-535bce170791.png)
When value is 1, for example:
![Screenshot 2021-04-14 at 18 15 35](https://user-images.githubusercontent.com/16876990/114736960-3c9f9280-9d4f-11eb-8a83-a2ebc5375b75.png)

**Versions**
`Flutter: 2.0`
`FlChart: 0.35.0`

```dart
double myValue;

return PieChartSectionData(
            color: const Color(0xff0293ee),
            value: myValue,
            title: myValue.toString(),
            radius: radius,
            titleStyle: TextStyle(
                fontSize: fontSize, fontWeight: FontWeight.bold, color: const Color(0xffffffff)),
          );
```
 
```dart
double myValue;

return PieChartSectionData(
            color: const Color(0xff0293ee),
            value: myValue != 0 ? myValue : null ,
            title: myValue.toString(),
            radius: radius,
            titleStyle: TextStyle(
                fontSize: fontSize, fontWeight: FontWeight.bold, color: const Color(0xffffffff)),
          );
```

**Comment by Mohdx** [comment]
![image](https://user-images.githubusercontent.com/51845593/114810059-e7f72880-9ddd-11eb-8ddd-d470771becf7.png)

**Comment by imaNNeoFighT** [comment]
Okay fair enough.
We remove the exception. And allow you to pass zero value.
Instead we remove those zero sections from output internally.

**Comment by mrdc** [comment]
Thanks! 👍

**Comment by imaNNeoFighT** [comment]
Hi, we have fixed it in [0.36.1](https://github.com/imaNNeoFighT/fl_chart/releases/tag/0.36.1).

Thanks for contributing!
